feature_engine/outliers/winsorizer.py

```diff
@@ -160,7 +160,7 @@ def transform(self, X: pd.DataFrame) -> pd.DataFrame:
         return X_out
 
     def get_feature_names_out(
-            self, input_features: Optional[List[Union[str, int]]] = None
+        self, input_features: Optional[List[Union[str, int]]] = None
```

Comment:
Same here: the only change is the indentation. Can we please remove this file from the PR?